components/nav/NavSideItem.vue

```diff
@@ -75,6 +78,41 @@ const noUserVisual = computed(() => isHydrated.value && props.userOnly && !curre
 </template>
 
 <style scoped>
+  .lozenge {
+    color: #fff;
+    font-size: 15px;
+    line-height: 20px;
+    max-width: 160px;
+    border-radius: 360px;
+    align-items: center;
+    background: #592ACB;
+    display: flex;
+    justify-content: center;
+    padding-right: 50px;
+
+    &:hover {
+      background:#7542E5;
+      color: #fff;
+    }
+  }
+
+  .dark .lozenge {
+    background: #CB9EFF;
+    color: #000;
+  }
+  @media screen and ( min-width: 640px ) {
+    .lozenge.item {
+      padding: 0.2rem;
+    }
+  }
+
+  @media screen and ( min-width: 1280px ) {
+    .lozenge.item {
+      padding-top: 12px;
+      padding-bottom: 12px;
+    }
+  }
+
```

Comment:
Just confirming on colors; are we going with the purple regardless of what they have the accent color set as?

<img width="1160" alt="image" src="https://github.com/MozillaSocial/elk/assets/1273879/f549daf1-e26b-4329-a004-ca25aa579713">

Reply:
++ we should definitely use the accent color

Reply:
This was the color that was specifically requested by design feedback but I can have that conversation. 

What is the color value that y'all would like me to use instead?

Reply:
Assuming `--c-primary` ?